includes/functions.php

```diff
@@ -76,9 +76,7 @@ function wc_gateway_ppec_log( $message ) {
  * @return bool Returns true if PayPal credit is supported
  */
 function wc_gateway_ppec_is_credit_supported() {
-	$base = wc_get_base_location();
-
-	return 'US' === $base['country'];
+	return 'USD' === get_option( 'woocommerce_currency', 'GBP' );
```

Comment:
This should use `get_woocommerce_currency()`.

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/425/commits/a7e1143d4b5113361a87eae9f5ba39acb6039a19